src/Pages/PassengerInfo/PassengerInfo.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import Accordian from './Accordian';
+import Accordian2 from './Accordian2';
+import Accordian3 from './Accordian3';
+
+import styled from 'styled-components';
+
+function PassengerInfo() {
+  // const [active, setActive] = useState('Title1');
+
+  return (
+    <>
+      <ContentsAll>
+        <Reservation>
+          <JourneyContent>
+            <span>여정 정보</span>
+            <JourneyInfo>
+              <p>편도 </p>
+            </JourneyInfo>
+          </JourneyContent>
+          <Passenger>
+            <span>승객(대표자) 정보</span>
+            <Accordian title="기본 정보" />
+            <Accordian2 title="연락처 정보" />
+            <Accordian3 title="유의사항" />
+          </Passenger>
+        </Reservation>
+        <Price>항공 운송료</Price>
+      </ContentsAll>
+    </>
+  );
+}
+const ContentsAll = styled.article`
+  display: flex;
+  flex-direction: row;
+  width: 1300px;
+  margin: 100px 0 0 300px;
+`;
+const Reservation = styled.div`
+  width: 1000px;
+  min-height: 100vh;
```

Comment:
```suggestion
}

const ContentsAll = styled.article`
  display: flex;
  flex-direction: row;
  width: 1300px;
  margin: 100px 0 0 300px;
`;

const Reservation = styled.div`
  width: 1000px;
  min-height: 100vh;
```
- 구분되는 요소들끼리는 한줄씩 띄워주세요
- flex-direction: row;는 기본 값이라서 안 넣어주셔도 될거 같습니다
- min-height 100vh가 어색해보이는데 꼭 필요한 값인가요?